### ISSUE TITLE
Compass should be inaccessible when hidden

### ISSUE THREAD

**Issue by 1ec5** [mapbox/mapbox-gl-native#4820]
The compass is invisible when it points north, so it should be inaccessible to VoiceOver in that state. Alternatively, it should remain visible but allow upward and downward swipes to rotate the map.

/ref #1496
/cc @friedbunny
